torch/_dynamo/variables/tensor.py

```diff
@@ -466,6 +467,10 @@ def create(cls, tx, proxy, sym_num, **options):
         if sym_num is None:
             sym_num = get_fake_value(proxy.node, tx)
         proxy.node.meta["example_value"] = sym_num
+
+        if isinstance(sym_num, (sympy.Integer, int)):
+            return ConstantVariable(sym_num)
```

Comment:
You want to coerce the sympy.Integer to int here, no?

Reply:
Potentially, I was kind of hoping to keep it a sympy for cases where we .value it into a new (potentially sympy again) variable after some operation, but experimenting on that led me down a useless rabbit hole, so yes, I think thats probably right!